src/components/views/game/game/scoreannouncing/AnnouncementModal.scss

```diff
@@ -2,6 +2,10 @@
 
 .announcement-modal {
 
+  .overlay {
```

Comment:
what does this do? if I change this value, nothing changes
If you want a black background consider something like this: 
![Background half black](https://user-images.githubusercontent.com/91155317/168080194-ea95ad44-bfd4-4b4b-9270-c62b1f91cd12.PNG)

Reply:
Hell yeah, I've searched for exactlly this line of code! Thanx for your help!